resources/zib/zib-wounds.DateOfLastDressingChange.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-wounds.DateOfLastDressingChange" />
+  <text>
+    <status value="empty" />
+    <div xmlns="http://www.w3.org/1999/xhtml">No narrative is provided for definitional resources. A human-readable rendering can be found in the implementation guide(s) where this resource is used.</div>
+  </text>
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-wounds.DateOfLastDressingChange" />
+  <name value="ZibwoundsDateOfLastDressingChange" />
+  <title value="zib wounds.DateOfLastDressingChange" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="Date on which the dressing was last changed." />
+  <purpose value="This Observation resource represents the DateOfLastDressingChange concept of the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) Wound v3.3 (2020)](https://zibs.nl/wiki/Wound-v3.3(2020EN)) and the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) BurnWound v3.4 (2020)](https://zibs.nl/wiki/Burnwound-v3.4(2020EN))." />

```

Comment:
This concept is also part of the zib PressureUlcer.

Reply:
Fixed